src/Engine/AudioHandler/AudioHandler.cpp

```diff
@@ -0,0 +1,53 @@
+#include <glog/logging.h>
+
+#include <SDL.h>
+#include <stdio.h>
+#include <string>
+
+#include "AudioHandler.h"
+
+
+AudioHandler::AudioHandler() {
+
+}
+AudioHandler::~AudioHandler() {
+}
+void AudioHandler::init() {
+    //TODO only load really important things that need early access Like... Splash screen music or w/e
+}
+Mix_Music* AudioHandler:: load_music(std::string path) {
+    Mix_Music* ret = Mix_LoadMUS(path.c_str());
+
+    if ( ret == NULL ) {
+        LOG(FATAL) << "Fatal error, could not load music: " << path.c_str() <<
+                   "Error: " << Mix_GetError() << " Exiting...";
+    }
+
+    return ret;
+}
+Mix_Chunk* AudioHandler:: load_chunk(std::string path) {
+    Mix_Chunk* ret = Mix_LoadWAV(path.c_str());
+
+    if ( ret == NULL ) {
+        LOG(FATAL) << "Fatal error, could not load chunk: " << path.c_str() <<
```

Comment:
This only for initialization?

Reply:
Ah, I did this wrong. Good catch. I should check if the chunk has been already loaded into the map first, and then if it isn't, then call `Mix_LoadWAV`. 

I think I had it like this because I was desperate to just get the thing working lol